.github/workflows/ci-testing-deploy.yml

```diff
@@ -302,6 +302,8 @@ jobs:
         uses: actions/setup-python@v5
         with:
           python-version: ${{ matrix.python }}
+          cache: "pip"
+          cache-dependency-path: "services/web/server/requirements/ci.txt"
       - name: install uv
```

Comment:
all the changes in this file are probably incorrect. master does not look like this.